yaks/README.md

```diff
@@ -552,6 +552,28 @@ yaks = Yaks.new do
 end
 ```
 
+Yaks also has support for respecting the `include` query parameter (e.g.
+`include=posts.comments,account`), which is a behaviour you can include in
+your mappers:
+
+```ruby
+require "yaks/behaviour/optional_includes"
+
+class UserMapper < Yaks::Mapper
+  include Yaks::Behaviour::OptionalIncludes
+
+  has_many :posts
+  has_one :account
+end
+
+# ...
+
```

Comment:
Thanks, I didn't know how to run Ataru. Instead of adding anything to `ataru_setup.rb`, I reused the existing objects.

Btw, how do you run RuboCop locally? I tried `bundle exec rake rubocop` from both `yaks/` and top-level directory, and it says that the task is missing.